### ISSUE TITLE
Cannot import `custom-electron-titlebar/main` using TypeScript's `import`

### ISSUE THREAD

**Issue by N3wSk1Y** [AlexTorresSk/custom-electron-titlebar#188]
**I can't import something from `custom-electron-titlebar/main` using TypeScript's `import`**

```ts
import { setupTitlebar, attachTitlebarToWindow } from "custom-electron-titlebar/main";
```
```ts
Cannot find module 'custom-electron-titlebar/main' or its corresponding type declarations.
```

**Comment by DerGoogler** [comment]
`import` it's not from TypeScript. It's from ES6-Syntax.

**Comment by Krelborn** [comment]
Same problem here. Using electron-forge project, but I upgraded to latest typescript compiler as older version didn't support the export field in the package.json. This fixed my import in of `custom-electron-titlebar` in preload but seeing the same error reported here with the main import.

**Comment by ErrorGamer2000** [comment]
The package is missing `dist/main.d.ts". Temporary fix is to put this in a `.d.ts`:
```ts
import type cetMain from "custom-electron-titlebar/dist/main/main";

declare module "custom-electron-titlebar/main" {
  const exports: cetMain;
  export default exports;
}
```

**Comment by luojinzhang** [comment]
Rename `main.ts` to `index.ts` and change the export to this maybe ?
```
import setupTitlebar from './setup-main';
import attachTitlebarToWindow from './attach-titlebar-to-window';

export { setupTitlebar, attachTitlebarToWindow };
```
then you can use
```
import {
  setupTitlebar,
  attachTitlebarToWindow,
} from 'addons/custom-electron-titlebar/main';
```

**Comment by kairauer** [comment]
I've just installed 4.1.2 and the issue still exists.

**Comment by AlexTorresSk** [comment]
> I've just installed 4.1.2 and the issue still exists.

Yes, checking is an error by file name
Fixed on 4.1.3

**Comment by digitaltim-de** [comment]
Same error here
